ephemeris/library_data_example.yaml

```diff
@@ -0,0 +1,9 @@
+libraries:
+    - name: library_name_test_1
+      files:
+        - https://zenodo.org/record/60520/files/dbSNP_138.hg19.vcf
+    - name: library_name_test_2
+      files:
```

Comment:
Interesting idea, but in this case we could automatically figure out the local file and do not need to distinguish between local and url, isn't it?

```
if os.path.exists():
    cp ...
else:
   wget ...
```
A folder would be an other idea, in this case I actually like the `files` and `folder` naming. What do you think?